__tests__/app.test.js

```diff
@@ -0,0 +1,62 @@
+const request = require("supertest");
+const app = require("../app.js");
+const connection = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/index.js");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  if (connection.end) connection.end();
+});
+
+describe("1. GET /api/health", () => {
+  test("status: 200, responds with server up and running message", () => {
+    return request(app)
+      .get("/api/health")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.message).toBe("Server up and running!");
+      });
+  });
+});
+
+describe("2. GET /api/topics", () => {
+  test("status: 200, responds with an array of topics of correct length", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toHaveLength(3);
+      });
+  });
+
+  test("status: 200, responds with an array of objects formatted correctly", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              description: expect.any(String),
+              slug: expect.any(String),
+            })
```

Comment:
you could merge these two tests to make your code a little more DRY, you can check the length and then have your forEach on the next line.